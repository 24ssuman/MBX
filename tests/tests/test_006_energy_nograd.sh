#!/bin/bash

filename=$(basename "$0")
filename="${filename%.*}"

../../bin/energy_wograd-test inputs/${filename}.nrg > output

if diff expected/${filename}.out output &> /dev/null ; then
  echo "TEST $filename PASSED" 
else
  echo "TEST $filename FAILED"
fi

rm -f CU.log output 
