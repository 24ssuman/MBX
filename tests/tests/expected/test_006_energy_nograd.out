Energies without gradients:
system[....0]=     3.4337616667e+04    kcal/mol
